deploy/crds/applicationmonitoring.integreatly.org_applicationmonitorings_crd.yaml

```diff
@@ -10,33 +10,83 @@ spec:
     plural: applicationmonitorings
     singular: applicationmonitoring
   scope: Namespaced
-  version: v1alpha1
+  subresources:
+    status: {}
   validation:
     openAPIV3Schema:
-      required: ["spec"]
+      description: ApplicationMonitoring is the Schema for the applicationmonitorings
+        API
       properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
         spec:
-          required: ["labelSelector"]
+          description: ApplicationMonitoringSpec defines the desired state of ApplicationMonitoring
           properties:
-            labelSelector:
+            additionalScrapeConfigSecretKey:
+              description: AdditionalScrapeConfigSecretKey is the key under which
+                additional scrape configs are stored within the secret
               type: string
-              minimum: 1
-              description: The selector used by the Prometheus and Grafana Operators to discover resources
             additionalScrapeConfigSecretName:
+              description: AdditionalScrapeConfigSecretName is the name of the secret
+                from which additional scrape configs will be passed to the prometheus
+                operator
               type: string
-              description: The name of the secret from which additional scrape configs will be passed to the prometheus operator
-            additionalScrapeConfigSecretKey:
+            alertmanagerInstanceNamespaces:
+              description: AlertmanagerInstanceNamespaces is a list of namespaces
+                to watch for alertmanager custom resources
+              type: string
+            labelSelector:
+              description: LabelSelector is a selector used by the Prometheus and
+                Grafana Operators to discover resources
+              minLength: 1
```

Comment:
Previous validation on this was `minimum: 1`. That seemed wrong, if I'm reading [this page](https://book.kubebuilder.io/reference/markers/crd-validation.html) correctly:

Minimum: "specifies the minimum numeric value that this field can have."
MinLength: "specifies the minimum length for this string."